po/fr.po

```diff
@@ -29,11 +29,11 @@ msgstr "Aucune Application trouvée"
 #: data/com.github.donadigo.appeditor.appdata.xml.in:10
 msgid ""
 "Edit application entries shown in application menu and their properties."
-msgstr ""
+msgstr "Modifiez les raccourcis et leurs propriétés dans le menu des l'applications."
```

Comment:
This should be "Modifiez les raccourcis et leurs propriétés dans le menu des applications."